projects/storefrontlib/src/lib/product/components/product-details/product-summary/product-summary.component.html

```diff
@@ -1,24 +1,42 @@
-<div class="cx-product-summary">
-  <div class="cx-product-summary__price cx-product-summary__price" aria-label="new item price">
-    {{product?.price?.formattedValue}}
-  </div>
-
-  <div class="cx-product-summary__description">
-    <p [innerHTML]="product?.summary"> </p>
-  </div>
+<div class="product-summary">
+  <ng-container *cxOutlet="outlets.TITLE">
+    <div class="item-title">
```

Comment:
Can we leave out this wrapper div? It would make it clear and more flexible (i.e. suppose customer like to move the product code to the end of the summary).

Reply:
Agreed, I didn't think about this case